public/locales/fr/homepage.json.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
+// For more information, check the common.json.ts file
+
+export default {
+  "ueBrowser.title": "Guide des UEs",
+  "ueBrowser.subtitle": "Recherche rapide sur tes matières préférées",
+  "ueBrowser.searchBar.placeholder": "Rechercher dans le guide des UEs",
+  "userBrowser.title": "Trombinoscope",
+  "userBrowser.subtitle": "L'endroit parfait pour trouver la bonne personne",
+  "userBrowser.searchBar.placeholder": "Rechercher dans le trombinoscope",
+  "dailyTimetable.title": "EdT journalier",
+  "dailyTimetable.subtitle": "Ton EdT du jour",
+  "todaysBirthdays.title": "Anniversaires",
+  "todaysBirthdays.subtitle": "Les annivs du jour",
+  "todaysBirthday.age": "{{age}} ans",
```

Comment:
Un ptit `s` pour que ce soit plus cohérent
```suggestion
  "todaysBirthdays.age": "{{age}} ans",
```